### ISSUE TITLE
it's not available to adjust the size of the country Container , please add it

### ISSUE THREAD

**Issue by mohamed3051215** [natintosh/intl_phone_number_input#244]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by cody1024d** [comment]
Despite the poor formatting of this issue, I'd also like a bit more control on the Selector size. For my use-case, I'd like to specifically reduce the padding, etc.

**Comment by tomaz-dugii** [comment]
Yes please, add an option to change height of the input or lets say the whole widget.
